### PR TITLE
Add a piu config example

### PR DESCRIPTION
**Pull request by avaczi**

### CODE REVIEW

File: components/piu.rst [hunk 1]
```diff
@@ -38,3 +38,10 @@ How to configure
 .. code-block:: bash
 
     $ cat ~/Library/Application\ Support/piu/piu.yaml
+
+An example configuration:
+
+.. code-block:: json
+
+{even_url: 'https://even.example.com', odd_host: 'odd-eu-west-1.example.com'}
```
Comment (hjacobs) [inline components/piu.rst]: you have to indent the code

Comment (avaczi) [reply components/piu.rst]: lost in transmission :(

File: components/piu.rst [hunk 2]
```diff
@@ -38,3 +38,10 @@ How to configure
 .. code-block:: bash
 
     $ cat ~/Library/Application\ Support/piu/piu.yaml
+
+An example configuration:
+
+.. code-block:: json
```
Comment (hjacobs) [inline components/piu.rst]: you also need to change `json` to `yaml`

Comment (avaczi) [reply components/piu.rst]: haha.  Now I am wondering why I was in an undefined state - a yaml file with a json structure inside?

Comment (hjacobs) [reply components/piu.rst]: JSON is always valid YAML :smile:

**Comment by hjacobs** [comment]
:+1:

**Comment by avaczi** [comment]
:+1:

**Pull request merged**